google-ads/src/main/java/com/google/ads/googleads/lib/callables/ExceptionTransformation.java

```diff
@@ -0,0 +1,7 @@
+package com.google.ads.googleads.lib.callables;
+
+import com.google.api.gax.rpc.ApiException;
+
+public interface ExceptionTransformation {
+  Throwable transform(ApiException throwable);
```

Comment:
`ApiException` is a `RuntimeException`, but here we are "escalating" it to a `Throwable`, which means any exception returned by this transformation needs to be caught and handled. Is that necessary?

Reply:
Good point - the GAX interfaces here all deal with throwables. However, the GAPIC interfaces actually don't expose checked exceptions, everything gets rethrown as an ApiException. I've switched to deal with Throwables instead, PTAL.